<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schedule</class>
 <widget class="QMainWindow" name="Schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTableWidget" name="tw_schedule">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>861</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>300</y>
          <width>491</width>
          <height>413</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>단체명</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_org"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>소속동</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_dong"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>시작일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="s_date_edit"/>
           </item>
           <item>
            <widget class="QTimeEdit" name="s_time_edit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>종료일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="e_date_edit"/>
           </item>
           <item>
            <widget class="QTimeEdit" name="e_time_edit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="le_end"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>담당자</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_manager"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>담당 연락처</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_man_phone"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>행사명</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="s_name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>행사 상세</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="s_detail"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>공개여부</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reg_sch_btn_2">
             <property name="text">
              <string>입력</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="s_update_btn">
             <property name="text">
              <string>수정</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_btn">
             <property name="text">
              <string>취소</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="s_delete_btn">
             <property name="text">
              <string>삭제</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="call_input_btn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>300</y>
          <width>88</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>입력창호출</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
